test/unicode-spec.js

```diff
@@ -80,6 +80,11 @@ describe("unicode", function() {
         expect`გამარჯობა`.not.toParse(strictSettings);
     });
 
+    it("should build Armenian both inside and outside \\text{}", function() {
+        expect`ԱԲԳաբգ`.toBuild(nonstrictSettings);
```

Comment:
Based on rendering on katex.org, I'd think this test was passing even before this change?
Would it be possible to add a test that was failing before?